tests/baselines/reference/typedefinition01.baseline.jsonc

```diff
@@ -1,17 +1,4 @@
 // === goToType ===
-// === /tests/cases/fourslash/server/a.ts ===
-// export class [|Foo|] {}
-
 // === /tests/cases/fourslash/server/b.ts ===
 // import n = require('./a');
-// var x/*GOTO TYPE*/ = new n.Foo();
-
-  // === Details ===
-  [
-   {
-    "containerKind": "",
-    "containerName": "",
-    "kind": "",
-    "name": ""
-   }
-  ]
\ No newline at end of file
+// var x/*GOTO TYPE*/ = new n.Foo();
```

Comment:
@DanielRosenwasser @RyanCavanaugh @jakebailey should the user go to type Foo from variable x?

Reply:
I think so?

Reply:
For [example](https://www.typescriptlang.org/play/?#code/MYGwhgzhAEBiD29oG8C+AoAbmATtAHtALzQB2ApgO5yIAUAlANxA), `x` won't point to `Foo` in this case.

```
class Foo {}
var /**/x = new Foo();
```